packages/tooling/fast-tooling/src/web-components/html-render-layer-inline-edit/html-render-layer-inline-edit.style.ts

```diff
@@ -1,6 +1,6 @@
 import { css } from "@microsoft/fast-element";
 
-export const HTMLRenderLayerInlineEditStyles = css`
+export const htmlRenderLayerInlineEditStyles = css`
```

Comment:
Is there a reason for renaming this? It doesn't match other variable declarations (HTMLRenderOriginatorId, HTMLRenderLayerInlineEditStyles, HTMLRenderLayerInlineEditTemplate). Is there a pattern for using title casing versus camel casing?

Reply:
Oh, looks like I just missed a few. I will take another pass at this afterwards but in future the consistency on naming should be that classes and interfaces are PascalCase and variables are camelCase. We may have missed a few but that has been the established pattern thus far.

Reply:
On second thought, I've included all naming changes in this PR since I've already made a few fixes.